pyaedt/edb_core/siwave.py

```diff
@@ -740,8 +740,24 @@ def create_dc_terminal(
         return self.create_pin_group_terminal(dc_source)
 
     @pyaedt_function_handler()
-    def create_exec_file(self, add_dc=False, add_ac=False, add_syz=False):
-        """Create an executable file."""
+    def create_exec_file(
+        self, add_dc=False, add_ac=False, add_syz=False, export_touchstone=False, touchstone_file_path=""
+    ):
+        """Create an executable file.
+        Parameters
```

Comment:
```suggestion
        
        Parameters
```
A blank line is needed before the Parameters section for proper rendering in the doc.